question28/js/modules/togglePassword.js

```diff
@@ -0,0 +1,11 @@
+export const togglePassword = (target) => {
+  if (target.nextElementSibling.type === 'password') {
+    target.nextElementSibling.type = 'text';
+    target.textContent = '非表示';
+    target.setAttribute('aria-label', 'パスワードを非表示にします');
+  } else {
+    target.nextElementSibling.type = 'password';
+    target.textContent = '表示';
+    target.setAttribute('aria-label', 'パスワードを表示にします');
+  }
+}
```

Comment:
When the button is pressed, the password is toggled on and off. The wording of the aria-label also switches.